cmdutils/state.go

```diff
@@ -26,6 +26,7 @@ type StateManager interface {
 	ImportState(io.Reader) error
 	ExportState(io.Writer) error
 	GetStore() (ds.Datastore, error)
+	GetOfflineState(ds.Datastore) (state.State, error)
```

Comment:
This method is not used anywhere outside this package. Why export it?

Reply:
It will be used.